code/script.js

```diff
@@ -1,47 +1,195 @@
 // All the DOM selectors stored as short variables
 const chat = document.getElementById('chat')
+const inputWrapper = document.getElementById('input-wrapper')
+const input = document.getElementById('input')
+const sendBtn = document.getElementById('send')
 
-// Global variables, if you need any, declared here
 
-// Functions declared here
+let questionNumber = 1
+
+
+const botReply = (msg) => {
+  showMessage(msg, 'bot')
+}
+
+const userReply = (msg) => {
+  showMessage(msg, 'user')
+}
 
-// This function will add a chat bubble in the correct place based on who the sender is
 const showMessage = (message, sender) => {
   if (sender === 'user') {
     chat.innerHTML += `
       <section class="user-msg">
         <div class="bubble user-bubble">
           <p>${message}</p>
         </div>
-        <img src="assets/user.png" alt="User" />  
+        <img src="assets/user.jpg" alt="User" />  
       </section>
     `
+
   } else if (sender === 'bot') {
     chat.innerHTML += `
       <section class="bot-msg">
-        <img src="assets/bot.png" alt="Bot" />
+        <img src="assets/bot.jpg" alt="Bot" />
         <div class="bubble bot-bubble">
           <p>${message}</p>
         </div>
       </section>
     `
   }
-  // This little thing makes the chat scroll to the last message when there are too many to be shown in the chat box
+
   chat.scrollTop = chat.scrollHeight
 }
 
+const nextQuestion = (message) => {
+  console.log('questionNumber', questionNumber)
+  if (questionNumber === 1) {
+    userReply(message)
+    input.value = ''
+    setTimeout(() => showCodeTypes(message), 1000)
+
+  } else if (questionNumber === 2) {
+    userReply(message)
+    setTimeout(() => showMenu(message), 1000)
+  } else if (questionNumber === 3) {
+
+    userReply(message)
+    setTimeout(() => showLike(message), 1000)
+
+  } else if (questionNumber === 4) {
+    userReply(message)
+    setTimeout(() => showFeeling(message), 1000)
+
+  } else {
+    userReply(message)
+    setTimeout(thankYou, 1000)
+  }
+}
+
 // Starts here
 const greeting = () => {
-  showMessage(`Hello there, What's your name?`, 'bot')
-  // Just to check it out, change 'bot' to 'user' here 👆
+  questionNumber = 1
+  botReply(`Hello! What's your name?`)
+}
+
+const showCodeTypes = (msg) => {
+  questionNumber++
+  botReply(
+    `Nice to meet you ${msg}. What do you think the programming-beginners liked most about this course?`
```

Comment:
You can change the name of the param to something more suiting, as long as you change it on row 78 as well:

```
const showCodeTypes = name => {
  questionNumber++
  botReply(
    `Nice to meet you ${name}. What do you think the programming-beginners liked most about this course?`
```